contracts/starknet/ProofReserve.cairo

```diff
@@ -134,12 +110,13 @@ func post_data_l2{
     asset_name_little: felt,
     address_owner_little: felt,
```

Comment:
why using the suffix `_little` here?

Reply:
because all the values should be in little endian representation

Reply:
just to add a notice